lib/sirsi_holding.rb

```diff
@@ -16,15 +16,20 @@ class SirsiHolding
   SKIPPED_LOCS = %w[BORROWDIR CDPSHADOW SHADOW SSRC-FIC-S STAFSHADOW TECHSHADOW WITHDRAWN].freeze
   TEMP_CALLNUM_PREFIX = 'XX('.freeze
 
-  attr_reader :id, :current_location, :home_location, :library, :scheme, :type, :barcode, :public_note, :course_reserves
+  attr_reader :id, :current_location, :home_location, :location, :status, :library, :scheme, :type, :barcode, :public_note, :course_reserves
 
+  # @param [String] location the folio code for location
+  # @param [Status] status the folio status
+  # @param [String] current_location Legacy parameter to be replaced by "location" and "status"
   # rubocop:disable Metrics/ParameterLists
-  def initialize(call_number:, home_location:, library:, barcode:, scheme: nil, current_location: nil,
+  def initialize(call_number:, home_location:, library:, barcode:, location:, status:, scheme: nil, current_location: nil,
```

Comment:
I've mostly been using the existing code that uses temporarylocation if it exists and searchworksTreatTemporaryLocationAsPermanentLocation is true, else permanent location otherwise effectiveLocation. This is the value that we've been using to map to home_location. I don't know what we want to call this. Is that not the value you want me to use?